gammapy/data/filters.py

```diff
@@ -97,3 +102,14 @@ def _filter_by_time(self, data):
     def copy(self):
         """Copy the `ObservationFilter` object."""
         return copy.deepcopy(self)
+
+    @staticmethod
+    def _check_filter_phase(event_filter):
```

Comment:
Why is this function hidden?

Reply:
I assume it had to be hidden because it is static and doesn't do anything interesting on it's own. 
But it could be public